src/typings/Types.ts

```diff
@@ -220,8 +221,11 @@ export interface RangeTarget {
   active: PrimitiveTarget;
   excludeAnchor: boolean;
   excludeActive: boolean;
+  rangeType: RangeType;
 }
 
+export type RangeType = "vertical" | "continuous";
```

Comment:
I think it's worth a comment to indicate what vertical / continuous mean

Reply:
done